opentelemetry/OpenTelemetry-Spans.md

```diff
@@ -0,0 +1,16 @@
+# OpenTelemetry Span Exporters
+
+For OpenTelemetry exporters use a `"collector.name"` of `"newrelic-opentelemetry-exporter"`
```

Comment:
Would you mind putting this in the ordered list?

Reply:
I can do that!

Reply:
done!